src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,34 @@
+package duke.command;
+
+import duke.data.exception.OutOfRangeException;
+import duke.data.task.Task;
+import duke.data.task.TaskList;
+import duke.message.Messages;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+public class DeleteCommand extends Command {
+    private final int Index;
```

Comment:
Avoid capitalizing the first letter of variables. Please use the camel case naming convention.